sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/autorest.md

```diff
@@ -0,0 +1,50 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+```yaml
+azure-arm: true
+csharp: true
+library-name: DefenderEasm
+namespace: Azure.ResourceManager.DefenderEasm
+require: https://github.com/Azure/azure-rest-api-specs/blob/23d88533ddfde4d1565a897fe95d42fb0d9333e5/specification/riskiq/resource-manager/readme.md
+tag: package-preview-2023-04
```

Comment:
Please comment out this line, the sdk should always generate from the default tag